docs/software-platforms/lsfr/creating-vcs.md

```diff
@@ -0,0 +1,104 @@
+# Creating virtual computers with Lightsail for Research
+
+While working on an [analysis module](../../contributing-to-analyses/analysis-modules/index.md), you can develop locally or on a virtual computer with Lightsail for Research (LSfR).
+
+Virtual computers provide access to a set amount of virtual CPUs, memory, and storage through a web browser rather than a physical machine.
```

Comment:
```suggestion
Virtual computers provide access to a set amount of virtual CPUs (vCPUs), memory, and storage through a web browser rather than a physical machine.
```